### DISCUSSION TITLE
如何修改证明中的结束符号？

### DISCUSSION THREAD

**Discussion by CalvinWaWang** [sjtug/SJTUThesis#986] [提问] [answered]
在使用sjtu的模版中，证明环境中的结束符号是实心方块。
一般来说应该是空心方块，请问如何修改？
![IMG_4497](https://github.com/sjtug/SJTUThesis/assets/33740878/6cc87002-0e75-4d42-8eca-66c4c9778c3d)

**Comment by LogCreative** [comment]
导言区添加：
```latex
\renewcommand{\QED}{\square}
```

**Reply by CalvinWaWang** [reply]
谢谢同学！

**Reply by CalvinWaWang** [reply]
不过，这样的话，当这个证明跨页的时候，会自动加上方块唉，这个如何解决呢？

![WechatIMG623](https://github.com/sjtug/SJTUThesis/assets/33740878/455a91ab-2fcd-4452-b29d-0a4eeb74716b)

**Reply by LogCreative** [reply, marked as answer]
这个一般认为是 `ntheorem` 宏包的bug，检查一下结束证明前有没有空行或注释；或者换用 `amsthm` 宏包。

https://github.com/sjtug/SJTUThesis/blob/2cbbecf929649aba722cd7f3d7fcafc52aec2d51/setup.tex#L190-L192

**Reply by CalvinWaWang** [reply]
非常感谢同学！
